Beta_Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py

```diff
@@ -193,35 +195,419 @@ def delete_indicator_command():
 
 
 def test():
-    get_list(demisto.params().get('blacklist_ip'))
+    """Perform API call to check that the API is accessible.
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    get_incidents_request(
+        {
+            'created_after': date.today(),
+            'state': 'open'
+        })
+    demisto.results('ok')
+
+
+# TRAP API
+def get_incident_field_context(incident):
+    """Parses the 'incident_fields' entry of the incident and returns it
+    Args:
+        incident (dict): The incident to parse
+    Returns:
+        list. The parsed incident fields list
+    """
+    incident_field_values = []
+    for incident_field in incident['incident_field_values']:
```

Comment:
incident.get('incident_field_values', []) will help avoid exception of accessing None variables